gaming-website/src/javaScript/games.json

```diff
@@ -0,0 +1,184 @@
+{
+  "payload": {
+    "gamesCover": [
+      {
+        "name": "Red Dead Redemption 2",
+        "price": "60",
+        "url": "../../public/assets/home-page/game-cover/red-dead-redemption-2.png"
+      },
+      {
+        "name": "Cyberpunk 2077",
+        "price": "60",
+        "url": "../../public/assets/home-page/game-cover/cyberpunk-2077.png"
```

Comment:
An easy way to reduce the size of the pr was to only introduce 2-3 entries in the json file plus the required images.